experience/resume.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title> Resume & Honors!</title>
+		<link rel="stylesheet" type="text/css" href="../css/style.css">
```

Comment:
@LuisLithgow The only thing left for the CSS to load is to rename the folder to all lowercase. Then it will work. Thats it!